tests/test_job_management.py

```diff
@@ -280,6 +279,10 @@ def get_expected_products_url(dict_param,
     dict_param_complete.pop("session_id", None)
     dict_param_complete.pop("job_id", None)
 
+    # to adapt to the dispatcher-app, and have it at the end of the url
```

Comment:
it's important somewhere that it is at the end?

Reply:
No, but the test would fail because the parameters are ordered differently from the way the dispatcher handles the `use_scws` parameter, ie added at the end of the dictionary, during the production of the url for the email.

Reply:
in general, python dictionaries were not really ordered until ~3.6. Even now they are ordered a bit unexpectedly (for performance). query parameter ordering is also sometimesambigious. Maybe it's better to avoid relying on the order and code..

Reply:
so maybe we should always order parameters in the url. To reduce ambiguity.

Reply:
I had to sort the keys of the dictionary that is then used to create the url

Reply:
> I had to sort the keys of the dictionary that is then used to create the url

It's good in any case, then URL is more deterministic. Same request will be expressed by the same URL.